evalml/pipelines/component_graph.py

```diff
@@ -319,13 +322,13 @@ def _consolidate_inputs(x_inputs, y_input, X, y):
             ww.DataTable, ww.DataColumn: The X and y transformed values to evaluate a component with
         """
         if len(x_inputs) == 0:
-            return_x = X
+            return_x = X.to_dataframe()
```

Comment:
Since we're now expecting `_compute_features` will be normalizing input to woodwork up the stack, we need to convert to pandas here. You can see that this X input is guaranteed to be woodwork by the code in `_compute_features`.

Reply:
Ah, woodwork conversions

Reply:
Haha yep. This is necessary because there's no concatenate feature in woodwork `0.0.11`. I'm pretty sure this is handled on @freddyaboulton 's accessor feature branch

Reply:
Yep, in the ww update branch we just do `return_x = X` hehe but fyi there still isn't a concatenate feature in ww